weatherApp/Models/Weather.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Weather.swift
+//  weatherApp
+//
+//  Created by Dario Nikolic on 03/08/2020.
+//  Copyright © 2020 Dario Nikolic. All rights reserved.
+//
+
+struct Weather {
+    let description: String
+    let icon: String
+    let id: Int
+    let main: String
+    var iconUrlString: String {
```

Comment:
If this value is always the same, it should be `let`

Reply:
iconUrlString is a computed property